src/Nri/Ui/Select/V8.elm

```diff
@@ -286,7 +310,14 @@ view label attributes =
             , valueToString = config.valueToString
             , defaultDisplayText = config.defaultDisplayText
             , isInError = isInError_
+            , disabled = disabled_
             }
+        , InputLabelInternal.view
```

Comment:
This is the funky bit. The only time the styles look off is when you check disabled or loading in the example app. It makes is look like the label is behind the border for a non-obvious to me reason (also im really rusty at css so maybe it is actually something simple...).

Flipping the order had no effect on the "normal" state of things as far as I can tell.